<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>St_Dialog</class>
 <widget class="QDialog" name="St_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameabel">
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="wcetLabel">
        <property name="text">
         <string>WCET</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="wcetEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="deadtimeLabel">
        <property name="text">
         <string>Deadlne</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="deadtimeEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="periodLabel">
        <property name="text">
         <string>Period</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="periodEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="jitterLabel">
        <property name="text">
         <string>Jitter</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="jitterEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="offsetLabel">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="offsetEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="isLabel">
        <property name="text">
         <string>isPeriodic(uncheck for sporadic)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="OkbuttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
